src/ATen/CMakeLists.txt

```diff
@@ -63,30 +65,31 @@ IF ($ENV{TH_BINARY_BUILD})
   SET(CMAKE_CXX_FLAGS "-static-libstdc++ ${CMAKE_CXX_FLAGS}")
 ENDIF()
 
-IF(NO_CUDA)
-  MESSAGE(STATUS "ignoring CUDA")
+IF(NOT NO_CUDA)
+  FIND_PACKAGE(CUDA 5.5)
+ENDIF()
+
+IF(NO_CUDA OR NOT CUDA_FOUND)
+	MESSAGE(STATUS "ignoring CUDA")
```

Comment:
Yeah - I guess my point is that the setup helper has already run through this process for us. So if it does not specify `NO_CUDA=1` and we can't find it, there is some error that has occurred. I tend to look at this from the perspective of ATen integrating with PyTorch, but its not a huge deal either way.